app/views/foreman_leapp/job_templates/leapp_remediation.erb

```diff
@@ -19,6 +19,9 @@ template_inputs:
   required: true
   options: "true\nfalse"
   default: "true"
+foreign_input_sets:
```

Comment:
@adamruzicka This is the first time I have seen it used. Do we have any user documentation on how & when to use it?

Reply:
The only other place where it's used is katello. [The old docs](https://theforeman.org/plugins/foreman_remote_execution/1.7/index.html) have something on it - search for "foreign input set"